scripts/cphd/testDir/test1.cpp

```diff
@@ -0,0 +1,26 @@
+/* =========================================================================
```

Comment:
What does this do?

Reply:
Was just testing my python script works with a real file. Was going to leave this out my bad.